 <script>
-import Electrical from "../lib/components/electrical.svelte";
-	import Retro from "../lib/components/retro.svelte";
-import Space from "../lib/components/space.svelte";
-
+	import Electrical from '../lib/components/electrical.svelte';
+	import Retro from '../lib/components/retro.svelte';
+	import Space from '../lib/components/space.svelte';
 </script>
 
-<h1>Creative coding</h1><button><a href="src/routes/retro.svelte">3e creative coding</a></button>
-<h2>Insane 90s space psychedelia list</h2><br>
+<h1>Creative coding</h1>
+<button><a href="/retro">3e creative coding</a></button>
+<h2>Insane 90s space psychedelia list</h2>
+<br />
 
 <Space />
 
-<br><h2>Electrical experimental escapist carousel</h2><br>
+<br />
+<h2>Electrical experimental escapist carousel</h2>
+<br />
 
 <Electrical />
 
 <Retro />
 
 <style>
-    button {
-        background-color: lightpink;
-        color: black;
-        border: none;
-        padding: 1em;
-        margin: 1em;
-        margin-left: 50em;
-        border-radius: 10px;
-        box-shadow: lightgrey 1px 1px 10px 1px;
-        font-weight: 600;
-    }
-
-    button:hover {
-        background-color: black;
-        color: lightpink;
-    }
-
-    a {
-        text-decoration: none;
-    }
-</style>
\ No newline at end of file
+	button {
+		background-color: lightpink;
+		color: black;
+		border: none;
+		padding: 1em;
+		margin: 1em;
+		margin-left: 50em;
+		border-radius: 10px;
+		box-shadow: lightgrey 1px 1px 10px 1px;
+		font-weight: 600;
+	}
+
+	button:hover {
+		background-color: black;
+		color: lightpink;
+	}
+
+	a {
+		text-decoration: none;
+	}
+</style>
diff --git a/src/routes/retro.svelte b/src/routes/retro.svelte
deleted file mode 100644
index cc5ba7c..0000000
--- a/src/routes/retro.svelte
+++ /dev/null
@@ -1,91 +0,0 @@
-<script>
-	import Retro from '../lib/components/retro.svelte';
-</script>
-
-<Retro />
-
-<section>
-	<svg width="100%" height="100%" xmlns="http://www.w3.org/2000/svg">
-		<path
-			d="M50 0 Q 100 -150 200 0 Q 300 150 400 0 Q 500 -150 600 0 Q 700 150 800 0 Q 900 -150 1000 0"
-			stroke="purple"
-			stroke-width="40"
-			fill="transparent"
-		/>
-		<path
-			d="M50 75 Q 100 -75 200 75 Q 300 225 400 75 Q 500 -75 600 75 Q 700 225 800 75 Q 900 -75 1000 75"
-			stroke="blue"
-			stroke-width="40"
-			fill="transparent"
-		/>
-		<path
-			d="M50 150 Q 100 0 200 150 Q 300 300 400 150 Q 500 0 600 150 Q 700 300 800 150 Q 900 0 1000 150"
-			stroke="green"
-			stroke-width="40"
-			fill="transparent"
-		/>
-		<path
-			d="M50 225 Q 100 75 200 225 Q 300 375 400 225 Q 500 75 600 225 Q 700 375 800 225 Q 900 75 1000 225"
-			stroke="yellow"
-			stroke-width="40"
-			fill="transparent"
-		/>
-		<path
-			d="M50 300 Q 100 150 200 300 Q 300 450 400 300 Q 500 150 600 300 Q 700 450 800 300 Q 900 150 1000 300"
-			stroke="orange"
-			stroke-width="40"
-			fill="transparent"
-		/>
-		<path
-			d="M50 375 Q 100 225 200 375 Q 300 525 400 375 Q 500 225 600 375 Q 700 525 800 375 Q 900 225 1000 375"
-			stroke="red"
-			stroke-width="40"
-			fill="transparent"
-		/>
-	</svg>
-
-	<img src="/assets/clipart120952.png" class="flower" alt="flower" />
-</section>
-
-<style>
-	* {
-		margin: 0;
-		padding: 0;
-	}
-
-	section {
-		background-color: rgb(255, 225, 169);
-		width: 100vw;
-		height: 100vh;
-		display: flex;
-		justify-content: center;
-		align-items: center;
-	}
-
-	.flower {
-		width: 10em;
-		position: fixed;
-	}
-
-	svg {
-		/* width: 100%;
- height: 100%; */
-		margin-top: 20rem;
-		margin-left: 20rem;
-	}
-
-	path {
-		stroke-dasharray: 150 2;
-		stroke-dashoffset: 1000;
-		animation: dash 60s linear alternate infinite;
-	}
-
-	@keyframes dash {
-		from {
-			stroke-dashoffset: 700;
-		}
-		to {
-			stroke-dashoffset: 0;
-		}
-	}
-</style>
